server/project/sbt-zookeeper-plugin/src/main/scala/SbtZookeeperPlugin.scala

```diff
@@ -107,7 +107,7 @@ object SbtZookeeperPlugin extends sbt.AutoPlugin{
 
     /** Settings **/
     zookeeperVersion := "3.4.7",
-    libraryDependencies += "org.apache.zookeeper" % "zookeeper" % zookeeperVersion.value,
+    libraryDependencies += "org.apache.zookeeper" % "zookeeper" % zookeeperVersion.value % "test",
```

Comment:
This still leaves us with jar hell in tests. not in the scope for this PR to fix it, but let's add a "TODO", to get it right.